python/packages/autogen-agentchat/src/autogen_agentchat/teams/_base_team.py

```diff
@@ -1,7 +1,16 @@
 from dataclasses import dataclass
+import logging
 from typing import List, Protocol
 
 from autogen_agentchat.agents._base_chat_agent import ChatMessage
+from autogen_core.application.logging import EVENT_LOGGER_NAME
+
+from ._utils import AgentChatLogHandler
+
+logger = logging.getLogger(EVENT_LOGGER_NAME + ".agentchatchat")
```

Comment:
Perhaps we can have a new agent chat constant `EVENT_LOGGER_NAME` which can be imported by user and the rest of the framework as

```python
from autogen_agentchat import EVENT_LOGGER_NAME
```

Reply:
The logging can be it's own module. `autogen_agentchat.logging`

Reply:
Moved to `autogen_agentchat.logging`